packages/grpc-js/src/server.ts

```diff
@@ -96,6 +96,7 @@ function getUnimplementedStatusResponse(
   return {
     code: Status.UNIMPLEMENTED,
     details: `The server does not implement the method ${methodName}`,
+    metadata: new Metadata(),
```

Comment:
The return type of this function is specifically written to not need this field. I don't think it is necessary to add it.